packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/gettingatoken.md

```diff
@@ -1,32 +1,50 @@
-This example shows part of the implementation of the `IDXClientDelegate` protocol functions the SDK calls for each response and for receiving the access token.
+Getting the authorization token takes two-steps. First, request a the token after the sign-in is successful. Then receive that token from the server. The access token is represented by a `Token` object from the `AuthFoundation` library that is a dependency for the Swift IDX SDK. You can store the token securely in the Keychain using `Credential`, another class from the `AuthFoundation` library.
```

Comment:
```suggestion
Getting the authorization token takes two steps. First, request an access token after the user signs in successfully. Then receive that token from the server. The token is represented by a `Token` object from the `AuthFoundation` library that is a dependency for the Swift IDX SDK. You can store the token securely in the Keychain using `Credential`, another class from the `AuthFoundation` library.
```

Reply:
Closing this comment as I rewrote things for the next comment